data/arjandhakal.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name":"Arjan Dhakal",
+  "type": "personal",
+  "bio": "Fiddling with Typescript and Rust",
+  "avatar": "https://github.com/arjandhakal.png",
+  "links":[
+    {
+      "name":"Github",
+      "url": "https://github.com/arjandhakal",
+      "icon": "FaGithub"
+    },
+    {
+      "name":"Blog Articles",
+      "url": "https://arjandhakal.hashnode.dev",
+      "icon": "FaBlogger"
```

Comment:
```suggestion
      "icon": "SiHashnode"
```

Reply:
As the blog site is Hashnode. So, I think hashnode icon will be better.

Reply:
Ah, good catch. Didn't know we had a icon for it.